docs/configuration.md

```diff
@@ -72,6 +72,58 @@ The configuration file after being read will be represented by `BdkConfig` class
 - The configuration for `app` is represented by `BdkExtAppConfig`.
 - The configuration for `ssl` is represented by `BdkSslConfig`.
 
+## BDK configuration example
+
+```yaml
+scheme: https
+host: localhost.symphony.com
+port: 8443
+
+pod:
+  host: devx1.symphony.com
+  port: 443
+
+agent:
+  context: agent
+
+keyManager:
+  host: sym-devx1-dev-chat-glb-1-ause1-all.symphony.com
```

Comment:
Not sensitive information, but remember that this is an open-source project. Could you put something less "technical" here?